exporter/clickhousetracesexporter/clickhouse_exporter.go

```diff
@@ -165,13 +165,28 @@ func populateOtherDimensions(attributes pdata.AttributeMap, span *Span) {
 			}
 			span.GRPCCode = strconv.FormatInt(statusInt, 10)
 		} else if k == "rpc.method" {
-			span.GRPCMethod = v.StringVal()
+			span.RPCMethod = v.StringVal()
+			system, found := attributes.Get("rpc.system")
+			if found && system.StringVal() == "grpc" {
+				span.GRPCMethod = v.StringVal()
+			}
+		} else if k == "rpc.service" {
+			span.RPCService = v.StringVal()
+		} else if k == "rpc.system" {
+			span.RPCSystem = v.StringVal()
 		}
 
 		return true
 
 	})
 
+	if _, ok := attributes.Get("rpc.grpc.status_code"); ok {
```

Comment:
Why not this snippet part of above if.. else.. statement (line 175 and onwards)?

Reply:
done